test/integration/users_edit_test.rb

```diff
@@ -0,0 +1,52 @@
+require "test_helper"
+
+class UsersEditTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test "an unauthorized user when they unsuccessfully edit their profiel gets
+  returned back to the profile edit view with an error" do
+    user = create(:user)
+    get edit_user_path(user)
+    assert_template 'users/edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.